packages/react-vanilla-components/__tests__/components/TelephoneInput.test.tsx

```diff
@@ -103,4 +103,30 @@ describe('Telephone-input', () => {
     expect(input).toHaveLength(1);
     expect(input[0]).toHaveAttribute('aria-describedby', `${f.id}__longdescription ${f.id}__shortdescription`)
   });
+
+  test('it should handle maxlength constraint', async () => {
+    const f = {
+      ...field,
+      maxLength: 5,
+      errorMessage: DEFAULT_ERROR_MESSAGE
+    };
+    let { renderResponse, element } = await helper(f);
+    const input = await renderResponse.findByPlaceholderText(f.placeholder);
+    userEvent.type(input, 'aaaaa');
+    expect(element.getState().value).toEqual('aaaaa');
+    expect(renderResponse.queryByText(DEFAULT_ERROR_MESSAGE)).toBeNull();
```

Comment:
Please add both positive / negative scenario in the unit test case

Reply:
Added!